vnext/Microsoft.ReactNative.IntegrationTests/ReactNativeHostTests.cpp

```diff
@@ -59,6 +61,14 @@ struct TestHostModule {
       TestCheck(hostObjGreeter2.getHostObject(rt) != nullptr);
     });
     TestCheck(jsiExecuted);
+
+    jsiExecuted = false;
+    ExecuteJsi(reactContext, [&](Runtime &rt) {
```

Comment:
>ExecuteJsi [](start = 4, length = 10)

(nit) is this synchrous execution?

Reply:
Yes, here it is synchronous because we run the Init method and JSI in JS queue.

---
In reply to: [542906082](https://github.com/microsoft/react-native-windows/pull/6745#discussion_r542906082) [](ancestors = 542906082)